site/package.json

```
@@ -30,12 +30,44 @@
     "gulp-rev-delete-original": "^0.2.3",
     "gulp-stylus": "^2.7.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-webserver": "^0.9.1"
+    "gulp-webserver": "^0.9.1",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "xo": "github:aslafy-z/xo#use-cosmiconfig"
   },
-  "author": "Swapnil Agarwal",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/swapagarwal/swag-for-dev/issues"
+  "scripts": {
+    "lint": "xo '**/*.js'",
```

```suggestion
    "lint": "xo",
```

XO already lints all JS files in all directories by default.